nncf/quantization/algorithms/min_max/onnx_backend.py

```diff
@@ -100,17 +101,25 @@ def mean_minmax_statistic_collector(use_per_sample_stats: bool,
                                                 window_size)
 
     @staticmethod
-    def get_initializer_value(model: onnx.ModelProto, initializer_name: str) -> np.ndarray:
-        for initializer in model.graph.initializer:
-            if initializer.name == initializer_name:
-                return onnx.numpy_helper.to_array(initializer)
-        raise RuntimeError(
-            'There is no initializer with the name {}'.format(initializer_name))
+    def get_weight_tensor(model: onnx.ModelProto, node: NNCFNode) -> Tuple[str, np.ndarray]:
+        onnx_graph = ONNXGraph(model)
+        node = onnx_graph.get_node_by_name(node.node_name)
+        return onnx_graph.get_weight_tensor(node)
+
+    @staticmethod
+    def get_weight_tensor_port_id(model: onnx.ModelProto, node: NNCFNode) -> int:
+        onnx_graph = ONNXGraph(model)
+        node = onnx_graph.get_node_by_name(node.node_name)
+        weight_tensor_name, _ = onnx_graph.get_weight_tensor(node)
+        for i, input_name in enumerate(node.input):
+            if input_name == weight_tensor_name:
+                return i
+        return -1
```

Comment:
Could you please explain what it means?

Reply:
It returns the index (port_id) where the weight of the node is located. If it wasn't find returns -1.

Reply:
Updated